test/recipes/80-test_ssl_old.t

```diff
@@ -466,10 +468,15 @@ sub testssl {
 	plan tests => $protocolciphersuitcount + scalar(@protocols);
 
 	foreach my $protocol (@protocols) {
+	    my $protoarg = "";
+	    $protoarg = "-ssl3" if $protocol eq "SSLv3";
+	    $protoarg = "-dtls1" if $protocol eq "DTLSv1";
+	    $protoarg = "-dtls12" if $protocol eq "DTLSv1.2";
```

Comment:
I would very much prefer if you made `protoarg` a list.  The reason is that the default value will give a 0 char argument string, and there's a risk that this becomes an actual empty argument in the call below, if it gets quoted.  It's better if the default is like this, as that will just disappear:

``` perl
	    my @protoarg = ();
```